test/baggage-span-processor.test.ts

```diff
@@ -0,0 +1,45 @@
+import { BaggageSpanProcessor } from '../src/baggage-span-processor';
```

Comment:
While the setup of the processor itself was different in the node distro, this specific test is pretty much copy-pasta but it seemed to still do what we need